test/imagekit/utils/formatter_test.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../spec_helper'
+require "rspec/autorun"
+
+RSpec.describe ImageKitIo::Utils::Formatter do
+  let(:subject) { described_class }
+
+  it "snake_to_camel_test" do
+    result=subject.snake_to_camel('my_snake_to_camel')
+    expect('mySnakeToCamel').to eq(result)
+  end
+
+  it "camel_to_snake_test" do
+    result=subject.camel_to_snake('mySnakeToCamel')
+    expect('my_snake_to_camel').to eq(result)
```

Comment:
Although technically same thing, but shouldn't it be written like this
```ruby
expect(result).to eq('my_snake_to_camel')
```

Reply:
Yeah, It should be the way you said but still, it works. The code was written by the previous developer, I didn't change anything besides the namespace here.